### PR TITLE
this is a feature

### PR DESCRIPTION
**Pull request by newproject25**
hello sajjad.your project is very good. i think this new feature make it more ### _interesting****_

### CODE REVIEW

File: file.js [hunk 1]
```diff
@@ -0,0 +1 @@
+let p = null;
```
Comment (sajjadmohammadi23) [inline file.js:1]: what is this?

Comment (newproject25) [reply file.js:1]: just null

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>p added</p>
```
Comment (sajjadmohammadi23) [inline index.html:10]: what about that?

Comment (sajjadmohammadi23) [reply index.html:10]: ?

Comment (sajjadmohammadi23) [reply index.html:10]: explaine that

**Review by sajjadmohammadi23 — COMMENTED** [review]
!!!!!!

File: index.html (hunk 2 above)
Comment (newproject25) [reply index.html:10]: just fun

**Comment by sajjadmohammadi23** [comment]
ok

**Pull request merged**